samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -20,6 +18,7 @@
 using SemanticKernel.Service.Model;
 using SemanticKernel.Service.Skills;
 using SemanticKernel.Service.Storage;
+using Directory = System.IO.Directory;
```

Comment:
Why do this rather than `using System.IO;`?

Reply:
less code/readability (btw, I don't do this manually, it's an automation code style fix in R#)

Reply:
`using System.IO;` is less code than `using Directory = System.IO.Directory;` :)

Reply:
sorry, I see what you mean, there's an ambiguous ref, the same Directory class in MsGraph lib

Reply:
> another Directory class in MsGraph lib

😦